ln-gateway/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 [dependencies]
 async-std = { version = "1.6.0", features = ["attributes", "tokio1"] }
 async-trait = "0.1.52"
-clightningrpc = { git = "https://github.com/elsirion/rust-clightning-rpc", rev = "28b0b062dfb7d758a3a0753c274febe4a7faa584" }
+cln-rpc = { git = "https://github.com/ElementsProject/lightning", rev = "151d0094354e20654f49a58aa4923b3c50faa4b7" }
```

Comment:
Ideally we'd have a crates.io dependency, asking the cln team if that will happen with the upcoming 0.11.0 release

Reply:
Can be updated now :tada: https://crates.io/crates/cln-rpc

Reply:
done